src/python/pants/backend/python/goals/export.py

```diff
@@ -102,6 +102,24 @@ class ExportPluginOptions:
         advanced=True,
     )
 
+    py_hermetic_scripts = BoolOption(
+        default=True,
+        help=softwrap(
+            """
+            When exporting a mutable virtualenv for a resolve, by default
+            modify console script shebang lines to make them "hermetic".
+            The shebang of hermetic console scripts uses the python args: `-sE`:
+
+            - `-s` skips inclusion of the user site-packages directoy,
+            - `-E` ignores all `PYTHON*` env vars like `PYTHONPATH`.
+
+            Set this to false if you need non-hermetic scripts with
+            simple python shebangs that respect vars like `PYTHONPATH`.
```

Comment:
Is it worth spelling this out an example? Someone might be unfamiliar with exactly what `PYTHONPATH` means but be expecting it to work, since it's the default behaviour (AIUI).

```suggestion
            simple python shebangs that respect vars like `PYTHONPATH`,
            to, for instance, allow importing modules installed globally
            on the system.
```

(Or something else like that, please choose a sensible/common/motivating example 😄 )

Reply:
I added an example of PyCharm injecting its debugging libs.